tutorials/Isotonic_Regression_And_Reliability_Diagrams.ipynb

```diff
@@ -45,7 +49,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For this tutorial, we generate synthetic forecast and observation data. To keep it simple, assume that these data are single-value mean forecasts of temperature and their corresponding observations over 100 locations for a single time (valid time)."
+    "For this tutorial, we generate synthetic forecast and observation data. To keep it simple, assume that these data are single-value forecasts of mean temperature and their corresponding observations over 100 locations for a single time (valid time)."
```

Comment:
Potentially change "mean" to be "expected". I think shifting the word later in the sentence may make some people think that it is a mean temperature across a period, rather than the mean of the predictive distribution (expected).

Reply:
No worries, will make that change.